### PR TITLE
Upgrad rapidsai-nightly Conda channel to  24.10

### PR DESCRIPTION
**Pull request by NvTimLiu**
rapidsai-nightly Conda channel had been upgraded to  24.10, 24.08 nightly Conda package is NOT available.

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

**Comment by NvTimLiu** [comment]
Build PASS locally

```
docker build -f ci/Dockerfile -t tim:test .
[+] Building 2.3s (8/8) FINISHED                                                                                                     
 => [internal] load .dockerignore                                                                                               0.0s
 => => transferring context: 2B                                                                                                 0.0s
 => [internal] load build definition from Dockerfile                                                                            0.0s
 => => transferring dockerfile: 1.63kB                                                                                          0.0s
 => [internal] load metadata for docker.io/nvidia/cuda:11.8.0-devel-ubuntu20.04                                                 2.2s
 => [1/4] FROM docker.io/nvidia/cuda:11.8.0-devel-ubuntu20.04@sha256:03681bbd11fea044ff3e3a5d65e190a6f935af30c56e03e740b27e456  0.0s
 => CACHED [2/4] RUN apt update -y     && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt install -y openjdk-8-jdk     && apt in  0.0s
 => CACHED [3/4] RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh     && mkdir /root/.co  0.0s
 => CACHED [4/4] RUN conda install -y -c rapidsai-nightly -c conda-forge -c nvidia cuml=24.10 python=3.9 cuda-version=11.8      0.0s
 => exporting to image                                                                                                          0.0s
 => => exporting layers                                                                                                         0.0s
 => => writing image sha256:1afdd65383c413e40fe3575a83d1856b15e8b45531bb2c025cb4b1a1fd83ec45                                    0.0s
 => => naming to docker.io/library/tim:test  
```

File: ci/Dockerfile [hunk 1]
```diff
@@ -37,6 +37,6 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86
     && conda config --set solver libmamba
 
 # install cuML
-ARG CUML_VER=24.08
+ARG CUML_VER=24.10
 RUN conda install -y -c rapidsai-nightly -c conda-forge -c nvidia cuml=$CUML_VER python=3.9 cuda-version=11.8 \
```
Comment (lijinf2) [inline ci/Dockerfile:41]: 24.08 is moved from "nightly" to "stable" release. 24.10 is under development for the next release so can be unstable. See rapids install selector for more info: https://docs.rapids.ai/install
 
Could you keep "24.08" and replace "rapidsai-nightly" with "rapidsai"?

Comment (eordentlich) [reply ci/Dockerfile:41]: Yes lets keep 24.10 on 24.08 rapidsai until it stabilizes a bit more and then switch to nightly.
But if nightly 24.08 is gone, can you also change this line to 24.08 rapidsai (i.e. not nightly) in the 24.08 branch?
https://github.com/NVIDIA/spark-rapids-ml/blob/branch-24.08/ci/Dockerfile#L41

Comment (NvTimLiu) [reply ci/Dockerfile:41]: Sounds good to me

BTW, this change targeted to spark-rapids-ml branch-24.10

**Review by eordentlich — APPROVED** [review]
Thanks!   I guess we need this also for 24.08 branch.
👍

**Comment by eordentlich** [comment]
Looks like this change got auto merged with https://github.com/NVIDIA/spark-rapids-ml/pull/709 when we fixed it in 24.08 so closing this PR.   Thanks @NvTimLiu  for catching this issue.

**Pull request closed without merging**